pages/linux/readpe.md

```diff
@@ -0,0 +1,28 @@
+# readpe
+
+> Displays information about PE files.
+> More information: <https://manpages.debian.org/testing/pev/readpe.1.en.html>.
+
+- Display all information about the PE file:
```

Comment:
```suggestion
- Display all information about a PE file:
```

Reply:
That page is quite old and doesn't really comply to our conventions yet. But when you say `the file` it soudns like there exists this one file, which is somehow special, whereas `a file` could be any file.

Reply:
OK, let's do this way then.